src/ucp/core/ucp_ep.h

```diff
@@ -60,8 +60,8 @@ enum {
     UCP_EP_FLAG_SOCKADDR_PARTIAL_ADDR  = UCS_BIT(21),/* DEBUG: Partial worker address was sent
                                                                to the remote peer when starting
                                                                connection establishment on this EP */
-    UCP_EP_FLAG_FLUSH_STATE_VALID      = UCS_BIT(22) /* DEBUG: flush_state is valid */
-
+    UCP_EP_FLAG_FLUSH_STATE_VALID      = UCS_BIT(22),/* DEBUG: flush_state is valid */
+    UCP_EP_FLAG_SOCKADDR_CLOSING       = UCS_BIT(23) /* DEBUG: close protocol is started */
```

Comment:
UCP_EP_FLAG_CLOSE_PROTO_VALID
 /* DEBUG: close protocol started and close_req is valid */